build/wix/build32.bat

```diff
@@ -0,0 +1,39 @@
+@echo off
```

Comment:
Oh okay then. You can leave them as-is now if you eventually plan to remove them.